### DISCUSSION TITLE
Welcome to bc-java Discussions!

### DISCUSSION THREAD

**Discussion by dghgit** [bcgit/bc-java#1094] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by theit** [comment]
Hi,

I'm a long-term Java user (>20y), and in our company we use BouncyCastle mainly for OpenPGP encryption and decryption of emnail messages. What I'd like to do with this community:
Well, get some insights in what is happening inside BC's code; troubleshooting when having problems; asking questions; trying to help others if possible when they have problems, ... :-)
